src/main/java/org/amahi/anywhere/activity/TestEditText.java

```diff
@@ -0,0 +1,47 @@
+package org.amahi.anywhere.activity;
+
+import android.content.Context;
+import android.support.design.widget.TextInputEditText;
+import android.util.AttributeSet;
+import android.view.KeyEvent;
+import android.view.View;
+import android.view.inputmethod.InputMethodManager;
+import android.widget.EditText;
+import android.widget.ImageView;
+
+public class TestEditText extends TextInputEditText {
```

Comment:
Please give it a proper name! 
`TestEditText` does not really signify anything.